SwiftUI/Model/SearchOperation/SearchOperation.swift

```diff
@@ -39,7 +39,7 @@ extension SearchOperation {
                           let data = html.data(using: .utf8) else { return }
                     result.snippet = try? NSAttributedString(
                         data: data,
-                        options: [.documentType: NSAttributedString.DocumentType.html],
+                        options: [.documentType: NSAttributedString.DocumentType.html, .characterEncoding: String.Encoding.utf8.rawValue],
```

Comment:
Thank you @HiroyasuNishiyama for this fix. Looks good to me, it actually fixes issues in other languages as well.
Please use the following formatting to resolve the lint issue:

```suggestion
                        options: [.documentType: NSAttributedString.DocumentType.html,
                                  .characterEncoding: String.Encoding.utf8.rawValue],
```